packages/react-select/src/NonceProvider.js

```diff
@@ -1,24 +1,28 @@
 // @flow
 import React, { Component, type Node } from 'react';
-import { CacheProvider } from '@emotion/core';
+import { CacheProvider } from '@emotion/react';
 import createCache from '@emotion/cache';
 import memoizeOne from 'memoize-one';
 
 type NonceProviderProps = {
   nonce: string,
   children: Node,
+  key: string,
```

Comment:
this should be something like `cacheKey` so it's not confused with the standard React `key` prop

```suggestion
  cacheKey: string,
```